client-course-schedulizer/public/dataInterfaces.tsx

```diff
@@ -0,0 +1,76 @@
+enum Semester {
+  Fall = "FA",
+  Interim = "IN", // TODO: Remove?
+  Spring = "SP",
+  Summer = "SU", // TODO: Is this a thing?
+}
+
+enum Half {
+  First = "first",
+  Full = "full",
+  Second = "second",
+}
+
+enum Day {
+  Friday = "F",
+  Monday = "M",
+  // TODO: Are weekends ever used? (Can't really hurt to have them)
+  Saturday = "S",
+  Sunday = "Su",
+  Thursday = "Th",
+  Tuesday = "T",
```

Comment:
I like the inclusion of the weekends just in case. Should we make Sunday and Thursday have a unique identifier? like N and R? And then we have a method to convert it to "Su" and "Th" when the schedule is output and returned to the registrar? Would it make our parsing easier?

Since it is an enum, parsing might not matter since we will be able to access the values via `Day.Thursday`

Reply:
I think parsing of our values wont matter much because they are stored as an array of these day strings (not as one string with multiple days). Parsing will come in for the CSVs we receive and we wont really be able to control the way they format Thursday (and Sunday), so I think the parsing "problem" will exist no matter how we store Thursday and Sunday.

Reply:
Yeah, that makes sense. Thanks!